metricbeat/module/redis/docker-compose.yml

```diff
@@ -2,10 +2,25 @@ version: '2.3'
 
 services:
   redis:
-    image: docker.elastic.co/integrations-ci/beats-redis:${REDIS_VERSION:-3.2.12}-1
+    image: docker.elastic.co/integrations-ci/beats-redis:${REDIS_VERSION:-3.2.12}-${REDIS_DISTRIBUTION:-oss}-1
     build:
       context: ./_meta
+      dockerfile: Dockerfile.${REDIS_DISTRIBUTION:-oss}
       args:
+        REDIS_DISTRIBUTION: ${REDIS_DISTRIBUTION:-oss}
         REDIS_VERSION: ${REDIS_VERSION:-3.2.12}
+    privileged: true
     ports:
       - 6379
+      - 8070
+  redis-enterprise:
```

Comment:
It'd be better to don't need to add a different service in the docker compose file, as we do with the several variants of mysql or with the different versions of nats.

In this case I think that we want to run the tests for OSS features on all versions, and we might need to add checks to skip the tests for enterprise features if they are not running against an enterprise version.

Reply:
Not relevant anymore.